ui/app/src/components/DeleteDialog/DeleteDialogUIBody.tsx

```diff
@@ -57,21 +63,70 @@ export function DeleteDialogUIBody(props: DeleteDialogContentUIProps) {
             selectedItems={selectedItems}
             disabled={isDisabled}
           />
-          <SelectionList
-            title={<FormattedMessage id="deleteDialog.dependentItems" defaultMessage="Dependent Items" />}
-            subtitle={<FormattedMessage id="deleteDialog.brokenItems" defaultMessage="Will have broken references" />}
-            emptyMessage={
-              <FormattedMessage id="deleteDialog.emptyDependentItems" defaultMessage="No dependent items" />
-            }
-            paths={dependentItems}
-            disabledPaths={disabledDependentItems}
-            displayItemTitle={false}
-            onSelectAllClicked={onSelectAllDependantClicked}
-            onItemClicked={onItemClicked}
-            selectedItems={selectedItems}
-            disabled={isDisabled}
-            onEditClick={onEditDependantClick}
-          />
+          <ListItem divider dense disableGutters={!Boolean(dependentItems)}>
+            <ListItemText
+              primary={
+                <Typography variant="subtitle1" component="span">
+                  <FormattedMessage id="deleteDialog.dependentItems" defaultMessage="Dependent Items" />
+                  {` • `}
+                  <FormattedMessage id="deleteDialog.brokenItems" defaultMessage="Will have broken references" />
+                </Typography>
+              }
+            />
+          </ListItem>
+          {dependentItems.length ? (
+            <List>
+              {dependentItems.map((path) => {
+                const labelId = `checkbox-list-label-${path}`;
+                return (
+                  <ListItem dense key={path}>
+                    <ListItemText
+                      id={labelId}
```

Comment:
What's this ID for? Remove if not required.

Reply:
not needed, removing